docs/subnets/subnet-hyperparameters.md

```diff
@@ -75,7 +75,8 @@ btcli sudo set
 
 ## max_weight_limit
 
-This is the maximum weight that can be set by a subnet validator for a subnet miner, expressed as a value between `0` and `65535`. This is a u16 (unsigned integer) type. 
+**Description**
```

Comment:
Representing a value between 0 and 1.0. with the weights of all miners in a subnet summing to 1.0